cmd/buckd/main.go

```diff
@@ -82,6 +82,10 @@ var (
 				Key:      "addr.powergate.api",
 				DefValue: "",
 			},
+			"ipnsRepublishSchedule": {
+				Key:      "ipns.republish.schedule",
```

Comment:
Nit picking here, but this could be `ipns.republish_schedule`, unless we think there will be other sub config values under `ipns.republish`